requirements-app.txt

```diff
@@ -44,14 +44,14 @@ awscli-cwlogs>=1.4.6,<1.5
 aws-embedded-metrics==1.0.7
 
 # Putting upgrade on hold due to new version introducing breaking changes
-Werkzeug==1.0.1
+Werkzeug==2.0.2
 MarkupSafe==2.0.1
 
 # Putting upgrade on hold due to v1.0.0 using sha512 instead of sha1 by default
 itsdangerous==0.24  # pyup: <1.0.0
 
-git+https://github.com/cds-snc/notifier-utils.git@46.5.0#egg=notifications-utils
-
+#git+https://github.com/cds-snc/notifier-utils.git@46.5.0#egg=notifications-utils
+git+https://github.com/cds-snc/notifier-utils.git@81e2fcdcb7c4d2ce3814af6db2ceb0ccd8f258f1
```

Comment:
We should probably tag a version of notification-utils to target the new version as described [in this document](https://docs.google.com/document/d/1CMCR4wiTisA3qZdkomuk0zriqeyVp-od5ZuUO95wATM/edit). Is that ok to do prior merging?